### PR TITLE
Added ability to pass inputs to InSpec shell using input file and cli

### LINKED ISSUE #4989: Add the ability to pass an input.yml file to the inspec shell
This would allow me to more consistently and easily test and work with controls that use shared inputs in my day to day devel work.

### PR DESCRIPTION
**Pull request by Nik08**
## Description

Added ability to pass inputs to InSpec shell using input file and cli
`inspec shell --depends <path> --input-file=<path> -t <target>`
`inspec shell --depends <path> --input a=[1,2,3] -t <target>`

## Related Issue

Fix #4989 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs-chef-io/content/inspec/shell.md [hunk 1]
```diff
@@ -230,3 +230,43 @@ $ inspec shell --format json -c 'describe file("/Users/test") do it { should exi
   }
 }
 ```
+
+## Running Chef InSpec Shell with inputs 
+
+Input option in shell subcommand would allow to more consistently and easily test and work with controls inside shell.
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/shell.md]: This is a bit rough. Perhaps:

The input options for the shell command allow you to provide values to profiles that are parameterized.  The allows you to work more consistently with these profiles when switching between `shell` and `exec` when using profiles that have inputs. For more details on inputs, see [the inputs reference](https://docs.chef.io/inspec/inputs/).

**Review by clintoncwolfe — APPROVED** [review]
Overall looks and works great! Needs a review from the docs team as there is quite a bit of doc update involved.

Good work!

**Comment by Nik08** [comment]
Thanks Clinton! :)

**Comment by IanMadd** [comment]
I made some edits. Can someone review my rewrites. Thanks!

**Comment by pburkholder** [comment]
Thank you!

**Pull request merged**